### ISSUE TITLE
Missing declaration file on v0.3.3 ?

### ISSUE THREAD

**Issue by ybiquitous** [chodorowicz/ts-debounce#16]
Hi.

It seems that `ts-debounce@0.3.3` has no declaration file (.d.ts). 😓 

### Reproduction

```ts
// index.ts
import { debounce } from "ts-debounce";

debounce();
```

```sh
$ npx tsc index.ts --strict
index.ts:1:26 - error TS7016: Could not find a declaration file for module 'ts-debounce'. '/Users/koba/tmp/aaaaa/node_modules/ts-debounce/dist/src/index.js' implicitly has an 'any' type.
  Try `npm install @types/ts-debounce` if it exists or add a new declaration (.d.ts) file containing `declare module 'ts-debounce';`

1 import { debounce } from "ts-debounce";
                           ~~~~~~~~~~~~~
```

I would be glad if you would check this problem.

**Comment by chodorowicz** [comment]
Hey @ybiquitous ! Yes, sorry, 0.3.3 was missing that file. I still need to figure out why it gets lost during some deploys. I've released 0.3.5 which has this file.
https://www.npmjs.com/package/ts-debounce/v/0.3.5

**Comment by ybiquitous** [comment]
@chodorowicz Thanks! I confirmed TypeScript compiler succeeds for my code! 😄
